.github/workflows/tests.yml

```
@@ -0,0 +1,64 @@
+name: Run Tests
+
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)